jobs/rootfs-builder.jpl

```diff
@@ -44,7 +44,22 @@ PIPELINE_VERSION
 @Library('kernelci') _
 import org.kernelci.util.Job
 
-def build(config, arch, pipeline_version, kci_core) {
+def build_buildroot(config, arch, pipeline_version, kci_core) {
+    dir(kci_core) {
+        sh(script: """\
+git clone https://github.com/kernelci/buildroot
```

Comment:
It looks like this should be done by the `kernelci.rootfs.buildroot` implementation rather than the Jenkins integration.  Then we won't need `build_buildroot` and `build_debos` in the Jenkins job.

Reply:
@mgalka  @gctucker , If we want to remove `git clone` from here, then its better to add this repo in buildroot docker image itself?

Reply:
I think that including clone of the repository in the Dockerfile may be the cleanest solution. Having said that we'll need to make sure that appropriate branches are used for `kernelci/buildroot` and `kernelci/staging-buildroot`.

Reply:
I guess we could have parameters in `rootfs-configs.yaml` with the buildroot git URL and revision, and the `kernelci.rootfs` code would handle setting up the source tree.  But maybe that should be done as a follow-up and we can just keep things in Jenkins as per this PR to get started.

Reply:
okay, Will look into `git clone` as separate PR either adding it on docker image or as part of `rootfs-configs.yml`.  I think this PR has all required changes now(?). @gctucker